correlation_rules/aws_usertakeoverviapasswordreset.yml

```diff
@@ -0,0 +1,24 @@
+AnalysisType: correlation_rule
+RuleID: "AWS.UserTakeoverViaPasswordReset"
+DisplayName: "AWS.UserTakeoverViaPasswordReset"
+Enabled: true
+Severity: High
+Detection:
+    - Sequence:
+        - ID: Password Reset
+          RuleID: AWS.CloudTrail.LoginProfileCreatedOrModified
+          MinMatchCount: 1
+          # TODO add a discovery step
+        - ID: Login
+          RuleID: AWS.Console.Login
+          MinMatchCount: 1
+      Transitions:
+        - ID: Match on IP
+          From: Password Reset
+          To: Login
+          Match:
+            - On: sourceIPAddress
```

Comment:
Should we not also be matching on the user id for the account as well? What about the case where an admin resets a user's password, then re-signs in to their own account?